src/storage/src/hummock/block_cache.rs

```diff
@@ -135,6 +136,7 @@ impl BlockCache {
                     Ok((block, len))
                 }
             })
+            .stack_trace("block_cache_lookup")
```

Comment:
I think this is not a "stack_trace". It's like a tracing span instead. I can do the renaming later when I introduce minitrace to the system. At that time, we might have something like `.rw_span` which creates both minitrace span and async stack trace info,